starfish/image/_filter/_base.py

```diff
@@ -5,7 +5,7 @@
 from starfish.pipeline.algorithmbase import AlgorithmBase
 from starfish.pipeline.pipelinecomponent import PipelineComponent
 from starfish.util import click
-
+from starfish.util.click.indirectparams import ImageStackParamType
```

Comment:
Did I miss the PR where ImageStackParamType gets defined?

Reply:
https://github.com/spacetx/starfish/pull/1124